app/controllers/users_controller.rb

```diff
@@ -9,15 +9,15 @@ def create
             render json: { errors: user.errors.full_messages }, status: :unprocessable_entity
         end
     end
-    def show
-        user = User.find_by(id: session[:user_id])
-        render json: user
+
+    def index
+        user = User.all
+        render json: user, status: :ok
     end
-    private
+
```

Comment:
Add the show, update, and destroy actions for a user

Reply:
Done the update and destroy

Reply:
Okay